packages/bpk-docs/src/layouts/links.js

```diff
@@ -326,6 +331,12 @@ const neoComponentsLinks = [
         children: 'Image',
         tags: ['web'],
       },
+      {
+        id: 'INFINITE_SCROLL',
+        route: routes.NEO_INFINITE_SCROLL,
+        children: 'Infinite Scroll',
```

Comment:
Can we change to sentence case i.e. 'Infinite scroll'?

Reply:
done